src/azure-cli-testsdk/azure/cli/testsdk/constants.py

```diff
@@ -5,3 +5,4 @@
 
 AUX_SUBSCRIPTION = '1c638cf4-608f-4ee6-b680-c329e824c3a8'
 AUX_TENANT = '72f988bf-86f1-41af-91ab-2d7cd011db47'
+AUX_TENANT_INFRASTRUCTURE = '6b085460-5f21-477e-ba44-1035046e9101'
```

Comment:
`AUX_TENANT_INFRASTRUCTURE` is used to store some online resources, please do not use it for testing